### PR TITLE
Add support for bigger columns range than A-Z but A-ZZ

### PR DESCRIPTION
**Pull request by foslund**
I experienced an error with a "Undefined offset: ?" because I had too many columns in the Google Sheet. So I've added support for bigger columns range A-ZZ.

Test:
- Add more columns untill you'll see "AA" in the header google sheet. And do a "translation_sheet:pull"

### CODE REVIEW

File: src/Sheet/TranslationsSheetCoordinates.php [hunk 1]
```diff
@@ -164,4 +164,14 @@ public function sourceFileColumnIndex()
     {
         return $this->getLocalesCount() + 4;
     }
+
+    public function returnRanges()
+    {
+        // Adding support for a bigger columns range fx. AA-ZZ columns
+        for ($x = 'A'; $x !== 'ZZ'; $x++) {
```
Comment (balping) [inline src/Sheet/TranslationsSheetCoordinates.php]: This is super expensive, going over 26*26+26 = 702 iterations every time (plus allocating that memory), just to select one single element from the array. I would suggest using a more efficient implementation like this:

https://github.com/PHPOffice/PhpSpreadsheet/blob/36acc372875c4d894dc093825ce4f62209db5a76/src/PhpSpreadsheet/Cell/Coordinate.php#L311

Comment (nbourguig) [reply src/Sheet/TranslationsSheetCoordinates.php]: Okay I can in fact see a performance issue here with supporting the full columns range. On the other hand, the script is only run locally, so I'm not sure If it's a big deal.

 Can you submit a request with the mentioned implementation ?

Comment (balping) [reply src/Sheet/TranslationsSheetCoordinates.php]: OK, I will

Comment (balping) [reply src/Sheet/TranslationsSheetCoordinates.php]: Here it is: #27

**Comment by nbourguig** [comment]
Fixed in #21

**Pull request closed without merging**